lib/scholars_archive/validators/graduation_year_length_validator.rb

```diff
@@ -0,0 +1,10 @@
+module ScholarsArchive::Validators
+  class GraduationYearLengthValidator < ActiveModel::Validator
+    def validate(record)
+      return if record.graduation_year.empty?
+      if record.graduation_year.length != 4
```

Comment:
This validates the length of a string.. what happens when someone enters "9999" or "FSCK"?

Since we expect `graduation_year` to be an actual year.. maybe use `DateTime` mojo to figure out of it's a year.